ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -427,7 +427,7 @@
                                        (.setTimeout js/window (fn [] (scroll/scroll-to-id "route-calendar-anchor")) 150)))
                  :row-selected?   #(= % selected-route)}
     [{:name "Reitti" :width "30%"
-      :read (juxt :route-short-name :route-long-name)
+      :read (juxt :gtfs/route-short-name :route-long-name)
```

Comment:
does this work?

Reply:
Will fix merge error, good catch. Merged this one too many times.